modules/directives/src/main/scala/scala/build/preprocessing/directives/DirectiveUtil.scala

```diff
@@ -38,18 +38,10 @@ object DirectiveUtil {
 
   def concatAllValues(
     scopedDirective: ScopedDirective
-  ): Seq[Positioned[String]] =
-    scopedDirective.directive.values.map {
-      case v: StringValue =>
-        val pos = position(v, scopedDirective.maybePath)
-        Positioned(pos, v.get)
-      case v: BooleanValue =>
-        val pos = position(v, scopedDirective.maybePath)
-        Positioned(pos, v.get.toString)
-      case v: EmptyValue =>
```

Comment:
no need to catch empty values here?

Reply:
I think we want to skip empty values, I use `collect`, so empty just won't be collected.